checker/src/main/java/org/checkerframework/checker/nullness/NullnessTransfer.java

```diff
@@ -324,9 +337,7 @@ public TransferResult<NullnessValue, NullnessStore> visitInstanceOf(
     @Override
     public TransferResult<NullnessValue, NullnessStore> visitMethodAccess(
             MethodAccessNode n, TransferInput<NullnessValue, NullnessStore> p) {
-        TransferResult<NullnessValue, NullnessStore> result = super.visitMethodAccess(n, p);
-        makeNonNull(result, n.getReceiver());
-        return result;
+        return super.visitMethodAccess(n, p);
```

Comment:
Do you think the new test failure is from this change?
If this method doesn't need to be overridden, please completely remove it. Otherwise, add a comment why we don't need to do anything.

Reply:
Yes, this line should not be deleted.